pipeline/config/exoscale/sc-config.yaml

```
@@ -0,0 +1,605 @@
+    daysToRetain: 7
+    schedule: 0 0 * * *
+    startingDeadlineSeconds: 200
+    resources:
+      requests:
+        cpu: 50m
+        memory: 100Mi
+      limits:
+        cpu: 250m
+        memory: 300Mi
+alerts:
+  alertTo: "null"
+  opsGenieHeartbeat:
+    enabled: false
+    url: https://api.eu.opsgenie.com/v2/heartbeats
+    name: set-me
+  slack:
+    channel: set-me
+  opsGenie:
+    apiUrl: https://api.eu.opsgenie.com
+externalTrafficPolicy:
+  local: false
+  whitelistRange:
+    global: 0.0.0.0/0
+    dex: false
+    kibana: false
+    elasticsearch: false
+    harbor: false
+    userGrafana: false
+    opsGrafana: false
+    prometheusWc: false
+nfsProvisioner:
+  server: not-used
+  path: /nfs
+  resources:
+    limits:
+      cpu: 100m
+      memory: 128Mi
+    requests:
+      cpu: 100m
+      memory: 128Mi
+  tolerations: []
+  affinity: {}
+  nodeSelector: {}
+ingressNginx:
+  controller:
+    # Chart deploys correctly but does not work with resourceRequests
+    resources: {}
+    # limits:
+    #   cpu: 100m
+    #   memory: 64Mi
+    # requests:
+    #   cpu: 100m
+    #   memory: 64Mi
+    tolerations:
+      - key: nodeType
+        operator: Exists
+        effect: NoSchedule
+    affinity: {}
+    nodeSelector: {}
+    config:
+      useProxyProtocol: false
+    useHostPort: true
+    service:
+      enabled: false
+      type: set-me
+      annotations: set-me
+    # Will add custom configuration options to Nginx https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/
+    additionalConfig: {}
+  defaultBackend:
+    # Chart deploys correctly but does not work with resourceRequests
+    resources: {}
+    # limits:
+    #   cpu: 100m
+    #   memory: 64Mi
+    # requests:
+    #   cpu: 100m
+    #   memory: 64Mi
+    tolerations:
+      - key: nodeType
+        operator: Equal
+        value: elastisys
+        effect: NoSchedule
+    affinity: {}
+    nodeSelector: {}
+velero:
+  enabled: true
+  tolerations: []
+  nodeSelector: {}
+  resources:
+    limits:
+      cpu: 200m
+      memory: 200Mi
+    requests:
+      cpu: 100m
+      memory: 100Mi
+  restic:
+    tolerations: []
+    resources:
+      limits:
+        cpu: 200m
+        memory: 200Mi
+      requests:
+        cpu: 100m
+        memory: 100Mi
+restore:
+  cluster: false
+  velero: false
+  veleroBackupName: latest
+issuers:
+  letsencrypt:
+    enabled: true
+    namespaces:
+      - dex
+      - elastic-system
+      - kube-system
+      - monitoring
+      - influxdb-prometheus
+      - harbor
+    prod:
+      email: letsencrypt@elastisys.com
+    staging:
+      email: letsencrypt@elastisys.com
+  extraIssuers: []
+certmanager:
+  resources: {}
+  nodeSelector: {}
+  tolerations: {}
+  affinity: {}
+  webhook:
+    resources: {}
+    nodeSelector: {}
+    tolerations: {}
+    affinity: {}
+  cainjector:
+    resources: {}
+    nodeSelector: {}
+    tolerations: {}
+    affinity: {}
+metricsServer:
+  enabled: true
```

[This PR](https://github.com/elastisys/compliantkubernetes-apps/pull/292) will probably be merged soon, so if you could add the new config added in that PR it'd be great.
```suggestion
  enabled: true
calicoAccountant:
  enabled: true
```